stdlib/3/multiprocessing/synchronize.pyi

```diff
@@ -0,0 +1,53 @@
+from typing import Callable, ContextManager, Optional, Union
+
+from multiprocessing.context import BaseContext
+import threading
+
+class Barrier(threading.Barrier):
+    def __init__(self,
+                 parties: int,
+                 action: Optional[Callable] = ...,
+                 timeout: Optional[float] = ...) -> None: ...
+
+class BoundedSemaphore(Semaphore):
```

Comment:
A bit confusing to have this above Semaphore (although that's legal in stubs).

Reply:
I tried to leave the classes in alphabetical order, but I can move it around if you'd like.

Reply:
Don't worry about it then. My general preference is to have them in the same order as the implementation file, but it doesn't really matter.